bind-service.html.md.erb

```diff
@@ -10,13 +10,13 @@ The <a href="../service-offerings/index.html" target="_blank">service marketplac
 Create the database:
 
 <pre class="terminal">
-$ cf create-service mongodb small my-mongodb
+$ cf create-service mongodb-2 small my-mongodb
 Creating service instance my-mongodb in org MyOrg / space MySpace as user@mydomain.com...
 OK
 
 Create in progress. Use 'cf services' or 'cf service my-mongodb' to check operation status.
 
-Attention: The plan `small` of service `mongodb` is not free.  The instance `my-mongodb` will incur a cost.  Contact your administrator if you think this is in error.
+Attention: The plan `small` of service `mongodb-2` is not free.  The instance `my-mongodb` will incur a cost.  Contact your administrator if you think this is in error.
```

Comment:
There are double spaces here and it should say 'an error': "Attention: The plan `small` of service `mongodb-2` is not free. The instance `my-mongodb` will incur a cost. Contact your administrator if you think this is an error."